### PR TITLE
Update screenocr extension

### PR DESCRIPTION
**Pull request by neo773**
## Description

This PR adds preferences to allow customizability and let user pick their primary language along with some other parameters
## Screencast
screenocr-1
screenocr-3

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by neo773** [comment]
@pernielsentikaer Can we get this merged?

**Comment by pernielsentikaer** [comment]
We are waiting for @joshauh46 check it too since he is the original author 🙂

**Comment by neo773** [comment]
> We are waiting for @joshauh46 check it too since he is the original author 🙂

@joshauh46 is me it’s my alt account I forgot to mention that. I’m moving away things from there to my main account.

**Comment by joshauh46** [comment]
> > We are waiting for @joshauh46 check it too since he is the original author 🙂
> 
> @joshauh46 is me it’s my alt account I forgot to mention that. I’m moving away things from there to my main account.
> 
> You can verify this by seeing the original commit for the extension it was by me

Hi, I'm neo773 replying from alt account

File: extensions/screenocr/package.json [hunk 1]
```diff
@@ -2,14 +2,113 @@
...
+      "title": "Language Correction",
+      "description": "Applies language correction during the recognition process. Disabling this property returns the raw recognition results, which provides performance benefits but less accurate results.",
+      "type": "checkbox",
+      "label": "Use Language Correction",
+      "default": true
+    }
+  ],
   "commands": [
+    {
+      "title": "Preferences",
```
[the remark this answers is not in the document]
Comment (neo773) [reply extensions/screenocr/package.json:107]: Agreed, what about “Select Recognition Languages” ?

[the remark this answers is not in the document]
Comment (neo773) [reply extensions/screenocr/package.json]: Done, just pushed the commit

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Comment by neo773** [comment]
@pernielsentikaer 
Not sure if this is the right place to discuss this, but anyway the raycast store web app markdown rendering has a bug where it does not add the associated style tag of the element. The button at the bottom looks enormous 
Screenshot 2023-05-22 at 8 24 54 PM
How it should look with the style tag
Screenshot 2023-05-22 at 8 26 23 PM

**Pull request merged**